public/app/features/profile/UserProfileEditPage.tsx

```diff
@@ -55,19 +63,92 @@ export function UserProfileEditPage({
   changeUserOrg,
   updateUserProfile,
 }: Props) {
+  const [queryParams, updateQueryParams] = useQueryParams();
+  const tabQueryParam = queryParams[TAB_QUERY_PARAM];
+  const [activeTab, setActiveTab] = useState<string>(
+    (typeof tabQueryParam === 'string' ? tabQueryParam : CORE_SETTINGS_TAB).toLowerCase()
+  );
+
   useMount(() => initUserProfilePage());
 
+  const extensionComponents = useMemo(() => {
+    const { extensions } = getPluginComponentExtensions({
+      extensionPointId: PluginExtensionPoints.UserProfileTab,
+      context: {},
+    });
+
+    return extensions;
+  }, []);
+
+  const groupedExtensionComponents = extensionComponents.reduce<Record<string, PluginExtensionComponent[]>>(
+    (acc, extension) => {
+      const { title } = extension;
+      if (acc[title]) {
+        acc[title].push(extension);
+      } else {
+        acc[title] = [extension];
+      }
+      return acc;
+    },
+    {}
+  );
+
+  const showTabs = extensionComponents.length > 0;
+  const tabs = [CORE_SETTINGS_TAB, ...keys(groupedExtensionComponents).map((title) => title)];
```

Comment:
also here is where we'd probably want to translate the `Core` label. instead of having an array of titles, think we're going to need something like `{title: string, id: string}`. for core, that might be:
```tsx
{
  id: CORE_SETTINGS_TAB,
  title: t('user-profile.tabs.core', 'Core')
}
```

for plugin extensions, maybe just 
```tsx
{
  id: title.toLowerCase(),
  title,
}
```

where `t` is the translate function from `app/core/internationalization` (or something like that). once you've got the translation function in, you can run `yarn i18n:extract` to generate the correct pseudo translation and keys in the other language files. see the docs [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md)

Reply:
added the translations, thanks for pointing this out 👍